### PR TITLE
Add a general procedure for Text builders.

### PR DESCRIPTION
**Pull request by JBYoshi**
I was just helping someone with migrating from formatting codes to `Text` objects. I wrote a quick guide on how to use `Text.builder()`, and he found it very helpful. This PR adds it to the docs so other people can make use of it.

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/0161ba4/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/0161ba4/plugin/text/text.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/696#issuecomment-392561247), not to the direct url to the preview._

File: source/plugin/text/text.rst [hunk 1]
```diff
@@ -40,6 +40,20 @@ The text builder interface allows for the creation of formatted text in a "build
     Read this `Wikipedia article <https://en.wikipedia.org/wiki/Builder_pattern>`__ for help understanding the purpose
     of the builder pattern in software design.
 
+General Procedure
+~~~~~~~~~~~~~~~~~
+
+1. Start with <code>Text.builder()</code>.
```
Comment (ST-DDT) [inline source/plugin/text/text.rst:46]: Please replace all `<code>Text</code>` blocks with ` ``Text`` ` blocks.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: source/plugin/text/text.rst (hunk 1 above)
Comment (ST-DDT) [reply source/plugin/text/text.rst:46]: As an alternative you could also use javadoc links : ``:javadoc:`Text#builder()` ``

**Comment by JBYoshi** [comment]
Closing because I just realized I had my knowledge totally wrong about how this works.

**Pull request closed without merging**